ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -36,7 +36,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: ocp-qe-perfscale-loaded-upgrade-aws
+- as: perfscale-aws-5nodes-loaded-upgrade-412to413to414-periodic
```

Comment:
here I think it would be ok to just put 412 to 414 because you have to go through 4.13 anyways

Reply:
true, will leave this anyway unless you feel strongly about it